srcpkgs/tlpui/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'tlpui'
+pkgname=tlpui
+version=1.5.0
+revision=1
+wrksrc="TLPUI-${pkgname}-${version}"
+build_style="python3-module"
+hostmakedepends="python3-setuptools"
+depends="tlp python3-gobject"
+short_desc="GTK user interface for TLP written in Python"
+maintainer="Visone Selektah <victortebar@gmail.com>"
+license="GPL-2.0-or-later"
+homepage="https://github.com/d4nj1/TLPUI"
+distfiles="https://github.com/d4nj1/TLPUI/archive/refs/tags/tlpui-${version}.tar.gz"
+checksum=37c31f88798c5dc2b7e77e49bcc2ef3d374abb93b2f903b2cd836fbcc5947b9a
+make_check=no
```

Comment:
gtk+3 in depends and `checkdepends="${depends}"` takes tests till `Gtk-WARNING **: cannot open display:` that is fixable too